tests/unit-tests/test_config_checks.py

```diff
@@ -435,6 +435,25 @@ def test_config_check_header_file(self):
         self.config['confluence_header_file'] = relbase + 'sample-header.tpl'
         self._try_config()
 
+    def test_config_check_confluence_table_width(self):
+        self.config['confluence_table_width'] = 123456
+        self._try_config()
+
+        self.config['confluence_table_width'] = '123456'
+        self._try_config()
+
+        self.config['confluence_table_width'] = 0
+        with self.assertRaises(ConfluenceConfigError):
+            self._try_config()
+
+        self.config['confluence_table_width'] = -123456
+        with self.assertRaises(ConfluenceConfigError):
+            self._try_config()
+
+        self.config['confluence_table_width'] = 'MyPage'
```

Comment:
Would be nice to have a value like `250mm` and verify no exception is generated.